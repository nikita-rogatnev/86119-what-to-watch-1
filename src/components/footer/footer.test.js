import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import Footer from "./footer";

Enzyme.configure({adapter: new Adapter()});

it(`Footer renders correctly`, () => {
  const tree = shallow(<Footer/>);

  expect(toJson(tree)).toMatchSnapshot();
});

