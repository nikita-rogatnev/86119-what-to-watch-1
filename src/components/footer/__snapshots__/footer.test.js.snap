// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="page-footer"
>
  <div
    className="logo"
  >
    <Link
      className="logo__link logo__link--light"
      to="/"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </Link>
  </div>
  <div
    className="copyright"
  >
    <p>
      Â© 2019 What to watch Ltd.
    </p>
  </div>
</footer>
`;
